### PR TITLE
Geometry module code review

### PR DESCRIPTION
**Pull request by simaQ**
关联 Issue https://github.com/antvis/g2/issues/1511

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Component style update
- [x] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] All tests pass and/or benchmarks are included
- [ ] Commit message follows commit guidelines
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: src/geometry/geometry.ts [hunk 1]
```diff
@@ -779,11 +800,10 @@ export default class Geometry {
...
 
   // 将数据归一化
-  private _normalizeValues(values, scale) {
+  private normalizeValues(values, scale) {
     let rst = [];
     if (_.isArray(values)) {
-      for (let i = 0, len = values.length; i < len; i += 1) {
-        const v = values[i];
+      for (const v of values) {
         rst.push(scale.scale(v));
```
Comment (hustcc) [inline src/geometry/geometry.ts]:
```ts
rst = values.map((v: any） => scale.scale(v));
```

File: src/geometry/geometry.ts [hunk 2]
```diff
@@ -579,11 +607,7 @@ export default class Geometry {
 
       attrCfg.scales = scales;
 
-      if (
-        _.indexOf(['color', 'size', 'shape', 'opacity'], attrType) !== -1 &&
-        scales.length === 1 &&
-        scales[0].type === 'identity'
-      ) {
+      if (['color', 'size', 'shape'].includes(attrType) && scales.length === 1 && scales[0].type === 'identity') {
```
Comment (hustcc) [inline src/geometry/geometry.ts]: GROUP_FILED

File: src/geometry/element.ts [hunk 3]
```diff
@@ -1,10 +1,9 @@
...
 import * as _ from '@antv/util';
+import { FIELD_ORIGIN } from '../constant';
 import { LooseObject, ShapeDrawCFG } from '../interface';
 import { ShapeFactory } from './interface';
 
-const FIELD_ORIGIN = '_origin';
-
 interface ElementCfg {
   /** 原始数据 */
   data: LooseObject;
```
Comment (hustcc) [inline src/geometry/element.ts]: data 数据相关的类型定义都用 Data 吧，虽然和 LooseObject 是一样的，但是代码字面含义上回好一些。

nit: LooseObject 的命名要不要跟简单一些，比如就叫 Obj，这个类型太常用了。

Comment (simaQ) [reply src/geometry/element.ts]: 命名尽量不要用简写吧

Comment (hustcc) [reply src/geometry/element.ts]: nit 可以不用修改的~只是觉得 LooseObject 太长了~

**Pull request merged**